lib/controller/prefs.js

```diff
@@ -0,0 +1,27 @@
+const prefsInput = require('../cli/input/prefs');
+const prefsOutput = require('../cli/output/prefs');
+const prefsModel = require('../model/prefs');
+const commandValidator = require('../utils/command-validator');
+
+const onPrefs = async (args, options, logger) => {
+  const isValidSession = commandValidator.validateSession();
+  if (!isValidSession) {
+    return;
+  }
+  let changePrefEvent = await prefsInput.getInput(args, options);
+  changePrefEvent = await prefsModel.storePrefs(changePrefEvent);
+  prefsOutput.sendOutput(changePrefEvent);
+}
+
+const addTo = (program) => {
+  program
+   .command('prefs', 'Show and change language and server preferences')
+   .argument('<preference>', 'Argument for prefs command', /^show|changelang|changeserver$/)
+   .option('--url <server_url>', 'New url for server')
```

Comment:
It is better to name this option as `--host`.

Also add an option for `--lang`

Reply:
Like we discussed, I am allowing only the change of server using flags. Language is changed using a prompt list. This is a better way because we can prevent the user from providing a random string, which is the case in version 1.